### ISSUE TITLE
Geeklog Installer Migrate Failes WHen Using BigDump and a compressed Backup

### ISSUE THREAD

**Issue by eSilverStrike** [Geeklog-Core/geeklog#984]
When I dumped one of my other dev geeklog databases (Using the built in Geeklog Backup which compressed it) and then tried to use it with migrate by selecting an existing backup from my backup directory. When I pressed the migrate button I got a blank screen and the following error in my php error log:
```
[07-Nov-2019 11:42:54 America/New_York] PHP Fatal error:  Uncaught splitbrain\PHPArchive\ArchiveCorruptedException: Header does not match it's checksum in C:\inetpub\GitHub\geeklog\system\vendor\splitbrain\php-archive\src\Tar.php:562
Stack trace:
#0 C:\inetpub\GitHub\geeklog\system\vendor\splitbrain\php-archive\src\Tar.php(92): splitbrain\PHPArchive\Tar->parseHeader('# Geeklog MySQL...')
#1 C:\inetpub\GitHub\geeklog\system\classes\unpacker.class.php(300): splitbrain\PHPArchive\Tar->contents()
#2 C:\inetpub\GitHub\geeklog\system\classes\unpacker.class.php(233): Unpacker->list_tar()
#3 C:\inetpub\GitHub\geeklog\public_html\admin\install\classes\installer.class.php(3347): Unpacker->getList()
#4 C:\inetpub\GitHub\geeklog\public_html\admin\install\classes\installer.class.php(3886): Installer->unpackFile('C:/inetpub/GitH...', 'geeklogtestinst...', '')
#5 C:\inetpub\GitHub\geeklog\public_html\admin\install\classes\installer.class.php(4642): Installer->migrateStep3()
#6 C:\inetpub\GitHub\geeklog\public_html\admin\install\classes\installer.class.php(5005): Installer->install in C:\inetpub\GitHub\geeklog\system\vendor\splitbrain\php-archive\src\Tar.php on line 
```562

**Comment by mystralkk** [comment]
Fixed with change set 42abfe6.

**Comment by eSilverStrike** [comment]
Tested and works.
